solution/ui/prototype/src/components/PDPart/PartSummary.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+    <div>
+        <h3>Part Summary</h3>
+        <p>Summary of the part</p>
+
+        <div>
+            <h3 style="display: inline">Featured Topics</h3>
+            <template v-if="!panel.length">
+                <v-btn style="float: right" @click="all" text
+                    >Expand All</v-btn
+                ></template
+            >
+            <template v-else>
+                <v-btn style="float: right" @click="hide" text
+                    >Hide All</v-btn
+                ></template
+            >
+                        <v-expansion-panels v-model="panel" multiple accordion>
+                <v-expansion-panel>
+                    <v-expansion-panel-content v-for="i in 3" :key="i">
+                        Featured topic  {{i}}</v-expansion-panel-content
+                    >
+                </v-expansion-panel>
+            </v-expansion-panels>
+        </div>
+        <h3 style="display: inline">
+            Part {{ part }} Federal Register Documents
+        </h3>
+        <template v-if="!panel.length">
+            <v-btn style="float: right" @click="all" text
+                >Expand All</v-btn
+            ></template
+        >
+        <template v-else>
+            <v-btn style="float: right" @click="hide" text
+                >Hide All</v-btn
+            ></template
+        >
+        
+        <v-expansion-panels v-model="panel" multiple accordion>
+            <v-expansion-panel>
+                <v-expansion-panel-content>
+                    <RelatedRules v-bind:title="title" v-bind:part="part"
+                /></v-expansion-panel-content>
+            </v-expansion-panel>
+        </v-expansion-panels>
+
+        <div>
+            <h3 style="display: inline">
+                Miscellaneous Resources not Tied to a Regulation
+            </h3>
+            <template v-if="!panel.length">
+                <v-btn style="float: right" @click="all" text
+                    >Expand All</v-btn
+                ></template
+            >
+            <template v-else>
+                <v-btn style="float: right" @click="hide" text
+                    >Hide All</v-btn
+                ></template
+            >
+            <v-expansion-panels v-model="panel" multiple accordion>
+                <v-expansion-panel>
+                    <v-expansion-panel-content v-for="i in 3" :key="i">
+                        Misc {{i}}</v-expansion-panel-content
+                    >
+                </v-expansion-panel>
+            </v-expansion-panels>
+        </div>
+        <br />
+        <v-btn color="#046791" class="white--text">View All Resources</v-btn>
+    </div>
+</template>
+<script>
+import RelatedRules from "../../../../regulations/js/src/components/RelatedRules.vue";
```

Comment:
Phil aliased in the existing site with this:
import RelatedRules from "legacy/js/src/components/RelatedRules.vue";